### DISCUSSION TITLE
Customizing

### DISCUSSION THREAD

**Discussion by thvi** [ProfSynapse/Professor-Synapse#6] [General]
I've added a few more rules that seemed relevant:
-reply in French
-provide examples
-explain why
-provide pros and cons
-provides sources

**Comment by ProfSynapse** [comment]
I would just be careful of "provides sources" since it will be like to hallucinate

**Reply by thvi** [reply]
Why? When I suggest "providing sources", I'm asking GPT to provide URL links.

**Reply by thvi** [reply]
Ah ok, I just understood your suspicion: GPT could provide sources that he himself invented.

**Reply by thvi** [reply]
I added "(URL links)" to force GPT to provide URLs only if sources are to be provided:
-provides sources (URL links).

That said, the few responses without the new "(URL links)" parameter that GPT has provided me with are URLs.

**Reply by ProfSynapse** [reply]
you can ask it to provide URL links in markdown (title)[url] so it'll hyperlink, but make sure you always check the links...it still often hallucinates.

**Reply by thvi** [reply]
Thank you! When you say hallucinations, do you mean outdated URL links?

**Reply by ProfSynapse** [reply]
My bad when I say hallucinations I mean that Large Language Models will commonly output complete fictions in a believable way. 

It's always important to evaluate the information yourself if you aren't already knowledgeable about it.

**Comment by ehzawad** [comment]
If you have a ChatGPT plus, you can avoid hallucinations in most cases. Just make sure that you're using https://chat.openai.com/?model=gpt-4-browsing (browse with Bing)
